packages/tasks/src/model-libraries.ts

```diff
@@ -586,6 +586,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		countDownloads: `path:"cvlm_llama2_tokenizer/tokenizer.model"`,
 		snippets: snippets.seed_story,
 	},
+	soloaudio: {
```

Comment:
Would be good to add a download count rule otherwise you won't really benefit from having this library "supported on HF".